src/test/scala/chiselTests/ToTargetSpec.scala

```diff
@@ -60,4 +94,26 @@ class ToTargetSpec extends ChiselFlatSpec with Utils {
       "You cannot access the .instanceName or .toTarget of non-hardware Data: 'tpe', in module 'Example'"
     )
   }
+
+  behavior.of(".toRelativeTarget")
+
+  it should "work with modules being elaborated within atModuleBodyEnd" in {
+    val chirrtl = ChiselStage.emitCHIRRTL(new RelativeOuterModule)
+
+    chirrtl should include("~RelativeOuterModule|RelativeOuterModule/inner:RelativeInnerModule>wire")
+  }
+
+  it should "default to the root module in the requested hierarchy" in {
+    val chirrtl = ChiselStage.emitCHIRRTL(new RelativeDefaultModule)
+
+    chirrtl should include("~RelativeDefaultModule|RelativeDefaultModule/inner:RelativeInnerModule>wire")
+  }
+
```

Comment:
Both of these tests have the top module as the one you're relative to (even though one uses `Some` vs. `None` as the argument). It would be good if one of the relative tests were relative to a module that is not the top.

Reply:
🤦 yeah, thanks.